include/wsrep/client_state.hpp

```diff
@@ -270,6 +270,16 @@ namespace wsrep
             return transaction_.start_transaction(id);
         }
 
+        /**
+         * Establish read view ID of the transaction.
+         */
```

Comment:
Please explain what the `gtid` parameter is for. Is it optional? If not, should be made a const reference.